poco_vendorConfig.cmake.in

```diff
@@ -6,7 +6,7 @@ set(_@PROJECT_NAME@_CONFIG_INCLUDED TRUE)
 
 # output package information
 if(NOT @PROJECT_NAME@_FIND_QUIETLY)
-  message(" - Found @PROJECT_NAME@: @PACKAGE_VERSION@ (${@PROJECT_NAME@_DIR})")
+  message(STATUS " - Found @PROJECT_NAME@: @PACKAGE_VERSION@ (${@PROJECT_NAME@_DIR})")
```

Comment:
done https://github.com/ros2/poco_vendor/pull/4/commits/2a61c14f5277b0a8814b440afc11ddddbafcca62, thanks for pointing it out